app/src/main/res/layout/popup_delete_list.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="vertical"
+    android:background="@color/transparent">
+
+    <TextView
+        android:id="@+id/placeholder_popup_delete_list"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/hori_small"
+        android:layout_marginTop="@dimen/vert_large"
```

Comment:
All other simple Alert Dialogs make use of around 10dp for vertical spacing at the top, start, and end. So you should use something like vert_medium for these margins.

Also, a small note, try keeping the equal margin values for both start and end for any element you use. This will help maintain symmetry.

Reply:
I have made the requested changes in my recent commit.